onnxruntime/core/session/inference_session.cc

```diff
@@ -1151,9 +1151,17 @@ common::Status InferenceSession::Initialize() {
     // need to keep the initializers if we're going to save the optimized model
     bool keep_initializers = !session_options_.optimized_model_filepath.empty();
 
+    // we only want to use the serialized session state in a minimal build.
+    // in a full build, even if we loaded from an ORT format model, optimizers may have run and changed the
+    // graph, which would result in it no longer matching the saved session state.
+    // as it's a full build we can create the SessionState as per usual using kernel lookups, so we can take the safe
+    // option and ignore the serialized session state.
+    const experimental::fbs::SessionState* serialized_session_state = nullptr;
```

Comment:
We're going to go with a different approach to handling ORT format models (PR that is part of should go out today). Is this change required for the PR to work or can it be reverted for now?

Reply:
We need this declaration for the regular build (non minimal build) to compile. I will remove the "auto*" in the following #ifdef block as Guoyu suggest.

---
In reply to: [521677655](https://github.com/microsoft/onnxruntime/pull/5771#discussion_r521677655) [](ancestors = 521677655)